common/app-starters-security-common/src/main/java/org/springframework/cloud/stream/app/security/common/SecurityCommonConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package org.springframework.cloud.stream.app.security.common;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+/**
+ * @author Christian Tzolov
+ * @author Artem Bilan
+ */
+@Configuration
+public class SecurityCommonConfiguration {
+
+
+	/**
+	 * The custom {@link WebSecurityConfigurerAdapter} to disable security in the application
```

Comment:
Please, revise this JavaDoc and the class name. 
This is not only about HTTP any more.
Plus the property is not a `http.enableSecurity` also...